assets/templates/account/password_change.html

```diff
@@ -1,14 +1,11 @@
-{% extends "base.html" %}
+{% extends 'account/settings/settings_authorisation.html' %}
 
 {% load i18n %}
 
-{% block content %}
-    <p><a href="{% url 'account_profile' %}">&larr; Back to profile</a></p>
-    <h1>{% trans "Change password" %}</h1>
-
+{% block details %}
     <form method="post" action="{% url 'account_change_password' %}">
         {% csrf_token %}
         {% include "snippets/bootstrap_form.html" %}
-        <button type="submit" name="action" class="btn btn-primary">{% trans "Change password" %}</button>
+        <p><button type="submit" name="user_submit" class="btn btn-primary btn-block">Save</button></p>
```

Comment:
Why are you changing the name attribute? I don't see that you use it elsewhere, so no reason to. You might confuse allauth.

Reply:
Reverted. Added btn-block in class